Sources/Trill.sln

```diff
@@ -43,20 +43,20 @@ Global
 		{3A2EB89B-70C7-4516-BB40-13248D5D2AB9}.Release|x64.Build.0 = Release|x64
 		{F88FBCF8-C33B-4089-8998-19E9CF901B99}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{F88FBCF8-C33B-4089-8998-19E9CF901B99}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{F88FBCF8-C33B-4089-8998-19E9CF901B99}.Debug|x64.ActiveCfg = Debug|x64
-		{F88FBCF8-C33B-4089-8998-19E9CF901B99}.Debug|x64.Build.0 = Debug|x64
+		{F88FBCF8-C33B-4089-8998-19E9CF901B99}.Debug|x64.ActiveCfg = Debug|Any CPU
```

Comment:
Why some of `Debug|x64` map to `Debug|Any CPU` and some map to just `x64` ?

Reply:
Hmm, I'm not sure, these changes were made automatically by VS. These two guids are the two remaining test projects, so I'm guessing .NET Core just targets Any CPU for both x64\AnyCPU

---
In reply to: [386604993](https://github.com/microsoft/Trill/pull/131#discussion_r386604993) [](ancestors = 386604993)